crates/frontend/src/components/table.rs

```diff
@@ -29,8 +30,35 @@ pub fn table(
                             .iter()
                             .filter(|column| !column.hidden)
                             .map(|column| {
-                                view! {
-                                    <th class="uppercase">{&column.name.replace('_', " ")}</th>
+                                let column_name = (&column.name).replace('_', " ");
+                                match column.sortable.clone() {
+                                    types::ColumnSortable::Yes {
+                                        sort_fn,
+                                        sort_by,
+                                        current,
+                                    } => {
+                                        view! {
+                                            <th
+                                                class="uppercase cursor-pointer"
+                                                on:click=move |_| sort_fn.call(())
+                                            >
+                                                {column_name}
+                                                    {
+                                                        if current {
+                                                            match sort_by {
+                                                                SortBy::Desc => view! { <i class="ri-arrow-down-s-line"></i> },
+                                                                SortBy::Asc => view! { <i class="ri-arrow-up-s-line"></i> },
+                                                            }
+                                                        } else {
+                                                            view! { <i class="ri-expand-up-down-line"></i> }
+                                                        }
```

Comment:
```suggestion
match (current, sort_by) {
(false, _) => ...
(true, Asc) =>
(true, Desc) => ... 
}
```

Reply:
No

Reply:
Why 😅 ?

Reply:
They are similar enough that this isn't justified